stubs/imagetransfers/stub.json

```diff
@@ -32,28 +32,76 @@
+    "times": 10000
+      }
+    ]
+  },
+  {
+    "path": "/ovirt-engine/api/imagetransfers/def456/cancel",
+    "method": "POST",
+    "responses": [
+      {
+        "responseCode": 200,
+    "times": 10000
+      }
+    ]
+  },
+  {
+    "path": "/ovirt-engine/api/imagetransfers/def456",
+    "method": "POST",
+    "responses": [
+      {
+        "responseBody": "<?xml version=\"1.0\" encoding=\"UTF-8\" standalone=\"yes\"?><image_transfer href=\"/ovirt-engine/api/imagetransfers/def456\" id=\"def456\"><direction>download</direction><format>raw</format><phase>transferring</phase><transfer_url>http://ovirt-imageio.konveyor-forklift:12345/images/cirros</transfer_url></image_transfer>",
+        "responseCode": 200,
+    "times": 10000
+      }
+    ]
+  },
+  {
+    "path": "/ovirt-engine/api/imagetransfers/def456",
     "method": "GET",
     "responses": [
       {
-        "responseBody": "<?xml version=\"1.0\" encoding=\"UTF-8\" standalone=\"yes\"?><image_transfer href=\"/ovirt-engine/api/imagetransfers/abc123\" id=\"abc123\"><direction>download</direction><format>raw</format><phase>transferring</phase><transfer_url>http://ovirt-imageio.konveyor-forklift:12345/images/cirros</transfer_url><host href=\"/ovirt-engine/api/hosts/2900d577-63a8-4ec9-acff-8493278346b1\" id=\"2900d577-63a8-4ec9-acff-8493278346b1\"/></image_transfer>",
+        "responseBody": "<?xml version=\"1.0\" encoding=\"UTF-8\" standalone=\"yes\"?><image_transfer href=\"/ovirt-engine/api/imagetransfers/def456\" id=\"def456\"><direction>download</direction><format>raw</format><phase>transferring</phase><transfer_url>http://ovirt-imageio.konveyor-forklift:12345/images/cirros</transfer_url><host href=\"/ovirt-engine/api/hosts/2900d577-63a8-4ec9-acff-8493278346b1\" id=\"2900d577-63a8-4ec9-acff-8493278346b1\"/></image_transfer>",
         "responseCode": 200,
         "times": 20
       },
       {
-        "responseBody": "<?xml version=\"1.0\" encoding=\"UTF-8\" standalone=\"yes\"?><image_transfer href=\"/ovirt-engine/api/imagetransfers/abc123\" id=\"abc123\"><direction>download</direction><format>raw</format><phase>finished_success</phase><transfer_url>http://ovirt-imageio.konveyor-forklift:12345/images/cirros</transfer_url><host href=\"/ovirt-engine/api/hosts/2900d577-63a8-4ec9-acff-8493278346b1\" id=\"2900d577-63a8-4ec9-acff-8493278346b1\"/></image_transfer>",
+        "responseBody": "<?xml version=\"1.0\" encoding=\"UTF-8\" standalone=\"yes\"?><image_transfer href=\"/ovirt-engine/api/imagetransfers/def456\" id=\"def456\"><direction>download</direction><format>raw</format><phase>finished_success</phase><transfer_url>http://ovirt-imageio.konveyor-forklift:12345/images/cirros</transfer_url><host href=\"/ovirt-engine/api/hosts/2900d577-63a8-4ec9-acff-8493278346b1\" id=\"2900d577-63a8-4ec9-acff-8493278346b1\"/></image_transfer>",
         "responseCode": 200,
+    "times": 10000
+      }
+    ]
+  },
+  {
+    "path": "/ovirt-engine/api/imagetransfers",
```

Comment:
@liranr23 I've added a new stub for different image transfer endpoints, but I'm not so sure about the post operation, we will need somehow distinguish between the secure and insecure flows to know which transfer to create. same here: https://github.com/kubev2v/fakeovirt/blob/5104cfaa997a345e0e69a9ad702e6d73d5e360dc/internal/app/handlers/static/static-handler.go#L120